pyproject.toml

```diff
@@ -25,6 +25,7 @@ keywords=["py_trees", "py-trees", "behaviour-trees"]
 [tool.poetry.dependencies]
 python = "^3.8"
 pydot = ">=1.4"
+py = "^1.11.0"
```

Comment:
had to add for newer PyTest versions: https://github.com/ionelmc/pytest-benchmark/issues/226